### PR TITLE
Feature: Vagrantfile: Increase video memory plus other performance optimizations

### PR DESCRIPTION
**Pull request by stephengtuggy**

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -46,120 +47,156 @@ vegastrike_assets_repository = "https://github.com/vegastrike/Assets-Production.
...
+                apt-get update
+                apt-get install -y ansible python-apt
+                apt-get install -y git
+                apt-get install -y "#{debian_desktop_environment}" virtualbox-guest-dkms virtualbox-guest-utils virtualbox-guest-x11
+                pushd /home/vagrant
+                if [ ! -d "Assets-Production" ]; then
+                su -c "git clone #{vegastrike_assets_repository}" vagrant
+                else
+                pushd "Assets-Production"
+                su -c "git pull" vagrant
```
Comment (BenjamenMeyer) [inline Vagrantfile:79]: Cool - nice catch!

File: Vagrantfile [hunk 2]
```diff
@@ -46,120 +47,156 @@ vegastrike_assets_repository = "https://github.com/vegastrike/Assets-Production.
...
-				reboot
-			SHELL
-		end
-	end
+    if system_status_focal then
+        config.vm.define "vegastrike_ubuntu_focal" do | vs_ubuntu_focal |
+            vs_ubuntu_focal.vm.box = "#{image_ubuntu_focal}"
+            vs_ubuntu_focal.vm.network "private_network", ip: "#{base_network_vb}.11"
+            vs_ubuntu_focal.vm.hostname = "vegastrike-ubuntu-focal"
+            vs_ubuntu_focal.vm.boot_timeout = 900
```
Comment (BenjamenMeyer) [inline Vagrantfile:93]: Would be good to make that a variable at the top.

**Review by BenjamenMeyer — APPROVED** [review]
would be nice to make the timeout use a variable at the top too for its value; but acceptable AS IS.

Also, thanks for catching the `su` usage so that the Assets Productions stuff gets done under the vagrant user. That was my intention...just missed the username. Thanks!

File: Vagrantfile (hunk 2 above)
Comment (stephengtuggy) [reply Vagrantfile:93]: Good point.

**Pull request merged**